cluster-provision/gocli/cmd/run.go

```diff
@@ -706,6 +733,20 @@ func run(cmd *cobra.Command, args []string) (retErr error) {
 		}
 	}
 
+	if realtimeSchedulingEnabled {
+		nodeName := nodeNameFromIndex(1)
+		success, err := docker.Exec(cli, nodeContainer(prefix, nodeName), []string{
+			"/bin/bash",
+			"-c",
+			"ssh.sh sudo sysctl kernel.sched_rt_runtime_us=-1",
```

Comment:
That won't store the value into sysctl.conf, right?
Maybe we should also do that, in case the node ever reboots?

Reply:
I don't think that the node will reboot during a run for realtime tests, but I can add the setting to `/etc/sysctl.d/`.

Reply:
Ok done: The value is now stored in `/etc/sysctl.d/realtime.conf` and loaded with `sysctl --system`. This change only applies  to k8s-1.22 since the functional tests will only run with this version onwards.